Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -374,6 +388,14 @@ public static bool RenderInputField(ConfigFieldAttribute configFieldDetails, boo
             });
         }
 
+        public static T RenderInputField<T>(ConfigFieldAttribute configFieldDetails, T value, float labelWidth, string tooltip = null) where T : Enum
+        {
+            return InputRendererWrapper<T>(configFieldDetails.Label, value, labelWidth, tooltip, (s, b, arg3, arg4) =>
+            {
+                return (T)EditorGUILayout.EnumFlagsField(CreateGUIContent(configFieldDetails.Label, tooltip), value, GUILayout.ExpandWidth(true));
+            });
+        }
```

Comment:
I think that until we have a need for this function, it should not be included. This is in context with other comments made in this review. \*This\* conversation can be marked as resolved so long as the others are discussed.

Reply:
I've "unresolved" this conversation because it appears that this function is still included in the changes.

Reply:
Oh, I must of missed one. Let's get 'em.

Reply:
Removed!